reference/best_practises/common_best_practises.rst

```diff
@@ -0,0 +1,26 @@
+Common Best Practises
+=====================
+
+The Akeneo PIM standard project comes with a default configuration to help you during the development of your project.
+
+
+What do you have to version?
+----------------------------
+
+* Do **NOT** version vendors
+* Do **NOT** modify the code in the vendors
+
+You will not benefit of dependencies patches otherwise.
```

Comment:
explain why. When running composer update, any file can be overriden (and will be)